go/consensus/tendermint/full/full.go

```diff
@@ -794,6 +794,13 @@ func (t *fullService) GetStatus(ctx context.Context) (*consensusAPI.Status, erro
 			return nil, fmt.Errorf("failed to fetch current block: %w", err)
 		}
 
+		// Latest block epoch.
+		epoch, err := t.beacon.GetEpoch(ctx, status.LatestHeight)
+		if err != nil {
+			return nil, fmt.Errorf("failed to fetch epoch: %w", err)
```

Comment:
Could this fail when there's no committed blocks? Maybe it should only query the epoch in case if successfully fetches the block above.

Reply:
Nice, this was actually caught by the early query E2E test :-)

Reply:
:grin: oh yeah i thought we'll fail early above if there's no committed block, missed the exception.